### DISCUSSION TITLE
Another Real-Time Transcription App

### DISCUSSION THREAD

**Discussion by davabase** [openai/whisper#631] [Show and tell]
I took my real-time concept from https://github.com/davabase/whisper_real_time and decided to turn it into a fully featured GUI.

https://user-images.githubusercontent.com/69365652/205431746-1ece6d20-85a6-4112-ba5a-ee050b67268c.mp4

You can check it out here: https://github.com/davabase/transcriber_app
There's a built executable for Windows in the releases as well.

**Comment by K0IN** [comment]
I would really like to see that for meetings and with multiple input devices (at the same time) like local mic and audio playback. So you can see live what you and the other people in the call said.

**Comment by thebug404** [comment]
This is amazing!!! 🤯
